XIVSlothCombo/Combos/PvE/AST.cs

```diff
@@ -215,11 +215,15 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                 if (((!AlternateMode && MaleficList.Contains(actionID)) ||
                     (AlternateMode && CombustList.ContainsKey(actionID)) &&
                     !InCombat()))
-
+                {
                     if (IsEnabled(CustomComboPreset.AST_DPS_AutoDraw) &&
                         ActionReady(OriginalHook(AstralDraw)) && (Gauge.DrawnCards.All(x => x is CardType.NONE) || (DrawnCard == CardType.NONE && Config.AST_ST_DPS_OverwriteCards)))
                         return OriginalHook(AstralDraw);
 
+                    if (IsEnabled(CustomComboPreset.AST_ST_DPS_EarthlyStar) &&
+                        ActionReady(EarthlyStar))
+                        return EarthlyStar;
```

Comment:
In my opinion this should be a sub-option of the Opener, primarily so we can note in the description that it should target your target, not your self.
> To be used in conjunction with Redirect/Reaction/etc set up to place Earthly Star at your `<target>`.

As it is now, you can have Reaction/Redirect/etc place Earthly Star under your `<focus>` or `<self>` no problem, but theoretically if it's in the opener it could not reach the boss with that setup (e.g. Alliance Raids).

---

As a sidenote, this note is why I did not add Star to the Opener when I added Earthly Star: I want to use `<self>` so that the star's healing effects are not lost when used on wall bosses (e.g. Endsinger, your Star can cover most of the arena, but not if placed at `<target>`).
I am not opposed to its addition at all, but in my opinion it MUST be noted that it may not work without `<target>` being your setting.